app/src/main/java/org/selfconference/android/session/Room.java

```diff
@@ -1,21 +1,27 @@
 package org.selfconference.android.session;
 
 import android.os.Parcelable;
+import android.support.annotation.NonNull;
 import com.google.auto.value.AutoValue;
 
 @AutoValue public abstract class Room implements Parcelable {
 
-  public static Room emptyRoom() {
-    return create(-1, "");
+  /** Returns a Null Object to represent missing room data. */
+  @NonNull public static Room nullRoom() {
+    return create(-1, "TBD");
```

Comment:
Yup, that's fine!

On March 30, 2016 at 1:59:25 PM, Macklin Underdown (notifications@github.com) wrote:

In app/src/main/java/org/selfconference/android/session/Room.java:

>  import com.google.auto.value.AutoValue;
>   
>  @AutoValue public abstract class Room implements Parcelable {
>   
> -  public static Room emptyRoom() {
> -    return create(-1, "");
> +  /** Returns a Null Object to represent missing room data. */
> +  @NonNull public static Room nullRoom() {
> +    return create(-1, "TBD");
@crebma are you cool on using "TBD" as the copy for a missing room? (see screenshot in PR description for where this gets rendered)

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub